src/controllers/expenseController.js

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable no-console */
+'use strict';
+
+const { ExpenseService } = require('../services/expenseService');
+const expenseService = new ExpenseService();
+const { Op } = require('sequelize');
+const userService = require('../services/userService');
+
+async function createExpense(req, res) {
+  try {
+    const { userId, spentAt, title, amount, category, note } = req.body;
+
+    if (!spentAt || !title || !amount || !userId) {
```

Comment:
I see you created a validator for another request. So, this could also use validators